### PR TITLE
Unfollow find inactives

### LINKED ISSUE #189: Find Inactives should provide an Unfollow button
From an ask:

> When you have the extension for finding inactives, can there be an option to unfollow them? just keep going through and unfollowing them on that pop up screen instead of opening up each blog? c:

### PR DESCRIPTION
**Pull request by ChuckL**
Added functionality to actually unfollow when find_inactives finds an inactive user.
Fixes #189.

### CODE REVIEW

File: Extensions/find_inactives.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //* TITLE Find Inactives **//
-//* VERSION 0.2.1 **//
+//* VERSION 0.3.1 **//
```
Comment (Wolvan) [inline Extensions/find_inactives.js]: this should be 0.3.0

Comment (nightpool) [inline Extensions/find_inactives.js]: 1.0.0 bump seems justified

File: Extensions/find_inactives.js [hunk 2]
```diff
@@ -260,10 +258,13 @@ XKit.extensions.find_inactives = new Object({
...
 		for (var i=0;i<XKit.extensions.find_inactives.retired_people_list.length;i++) {
 
-			m_html = m_html + "<div data-url=\"http://" + XKit.extensions.find_inactives.retired_people_list[i].username + ".tumblr.com/\" class=\"find-inactives-blog\">" +
+			m_html = m_html + '<div class="find-inactives-blog">' +
+				'<div class="find-inactives-blog-link" data-url="http://' + XKit.extensions.find_inactives.retired_people_list[i].username + '.tumblr.com/">' +
 			"<div class=\"name\">" + XKit.extensions.find_inactives.retired_people_list[i].username + "</div>" +
 			"<div class=\"days\">" + XKit.extensions.find_inactives.retired_people_list[i].days + "</div>" +
 			"<img src=\"http://api.tumblr.com/v2/blog/" + XKit.extensions.find_inactives.retired_people_list[i].username + ".tumblr.com/avatar/512\" class=\"avatar\">" +
+				'</div>'+
+			'<div style="float:right; margin-top: 10px; margin-right: 6px;">' + XKit.extensions.find_inactives.retired_people_list[i].controls +'</div>' +
```
Comment (nightpool) [inline Extensions/find_inactives.js]: prefer CSS styles over inline ones

File: Extensions/find_inactives.js [hunk 3]
```diff
@@ -283,6 +284,22 @@ XKit.extensions.find_inactives = new Object({
 
 		});
 
+		$(".unfollow_button").bind("click", function() {
+			console.log(this);
+			var unfollowButton = $(this);
+			var name = $(this).attr("data-name");
```
Comment (nightpool) [inline Extensions/find_inactives.js]: use var unfollowButton instead of $(this) to avoid jquery variable initialization overhead

File: Extensions/find_inactives.js [hunk 4]
```diff
@@ -283,6 +284,22 @@ XKit.extensions.find_inactives = new Object({
 
 		});
 
+		$(".unfollow_button").bind("click", function() {
+			console.log(this);
```
Comment (nightpool) [inline Extensions/find_inactives.js]: remove debug log statement

File: Extensions/find_inactives.js [hunk 5]
```diff
@@ -290,7 +307,7 @@ XKit.extensions.find_inactives = new Object({
 
 		}
 
-		$(".find-inactives-blog").click(function() {
+		$(".find-inactives-blog-link").click(function() {
```
Comment (nightpool) [inline Extensions/find_inactives.js]: why is this changed? doesn't this break clicking on the username and the days inactive?

File: Extensions/find_inactives.js [hunk 6]
```diff
@@ -283,6 +284,22 @@ XKit.extensions.find_inactives = new Object({
 
 		});
 
+		$(".unfollow_button").bind("click", function() {
```
Comment (nightpool) [inline Extensions/find_inactives.js]: a little confused about why we have our own click handler here. shouldn't tumblr's click handler for unfollow-blog handle this? is there a way to make it handle this? that would be preferable to duplicating the code

File: Extensions/find_inactives.js (hunk 5 above)
Comment (Wolvan) [inline Extensions/find_inactives.js]: Tested it, it doesn't seem to break. The click should open the blog, right? That's what it does

Comment (nightpool) [inline Extensions/find_inactives.js]: right, but its just a smaller target then it used to be

Comment (Wolvan) [inline Extensions/find_inactives.js]: I don't see the problem then, I guess it would have opened a new tab when you clicked unfollow otherwise. Personally I always clicked on the blog name anyways

**Comment by nightpool** [comment]
@ChuckL ever come back to this? we get asks about it in support pretty frequently tbh.

**Comment by ChuckL** [comment]
Sorry, life stuff is happening.
I'll dedicate a few spare cycles Saturday to touch it up as well as #692

**Comment by nightpool** [comment]
cool. no biggie—i'm super swamped too

**Comment by StanfordAlan** [comment]
Does this no longer work? I cannot see the buttons in my install from this evening on Firefox 49.0.1 on Mac 10.12. I have a huge number of inactive blogs I'm following, and while each one shows up in the filtered list, there's no way I'm finding to unsubscribe except to click on each blog's title and hit Unfollow. 

I'm having a bit of trouble following the post on the topic, as there are a few, but I thought it was committed a year ago.

Thanks.

**Comment by nightpool** [comment]
@StanfordAlan this was a pull request—it never actually got merged. I'll look into getting it up to date.
